src/goto-programs/adjust_float_expressions.h

```diff
@@ -9,18 +9,16 @@ Author: Daniel Kroening, kroening@kroening.com
 /// \file
 /// Symbolic Execution
 
-#ifndef CPROVER_GOTO_SYMEX_ADJUST_FLOAT_EXPRESSIONS_H
-#define CPROVER_GOTO_SYMEX_ADJUST_FLOAT_EXPRESSIONS_H
+#ifndef CPROVER_GOTO_PROGRAMS_ADJUST_FLOAT_EXPRESSIONS_H
+#define CPROVER_GOTO_PROGRAMS_ADJUST_FLOAT_EXPRESSIONS_H
 
-#include <goto-programs/goto_functions.h>
+#include "goto_functions.h"
 
 class exprt;
 class namespacet;
 class goto_modelt;
 
-void adjust_float_expressions(
-  exprt &expr,
-  const namespacet &ns);
+void adjust_float_expressions(exprt &expr, const namespacet &ns);
```

Comment:
As above, I'd advise against reformatting.

Reply:
Hi @tautschnig. How strongly do you object to the reformatting of these files? The rationale for the changes is that since they are getting moved anyway, we might as well just reformat them to the new automatically enforced style. This is a notion I find myself agreeing with.

Why is it a problem if we do it? If it's truly a problem, how do you suggest we rectify the situation?

Reply:
There are three contexts of why this is a problem:
1) Reviewing is somewhere between impossible and impractical.
2) If anyone else has patches against those files, whether currently under review in some PR or completely out-of-tree, then it any attempt to rebase or merge will require a rewrite of the patches. At bare minimum it is necessary to split the move of the file and adjustments to formatting into separate commits.
3) Trying to track down the rationale of some lines of code via `git blame` becomes a lot harder.

And all that with no known benefit on the plus side.

Reply:
That's fair, I'll undo